GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -310,6 +313,7 @@ gdjs.SpriteRuntimeObject.thisIsARuntimeObjectConstructor = "Sprite"; //Notify gd
 
 /**
  * Initialize the extra parameters that could be set for an instance.
+ * @param {{numberProperties: Array<{name: string, value: any}>, customSize: {width: number, height: number}}} initialInstanceData The extra parameters
```

Comment:
value is guaranteed to be a number in numberProperties, so

`Array<{name: string, value: number}`